src/components/events/event_new/EventNewForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Form from 'react-bootstrap/Form';
+import { Button } from 'react-bootstrap';
```

Comment:
Form and Button can be imported together ;)

Reply:
Haha nice catch.